kroxylicious-kms-tls-support/src/main/java/io/kroxylicious/proxy/tls/InsecureTrustManager.java

```diff
@@ -6,16 +6,20 @@
 
 package io.kroxylicious.proxy.tls;
 
+import java.net.Socket;
+import java.security.cert.CertificateException;
 import java.security.cert.X509Certificate;
 
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.X509ExtendedTrustManager;
 import javax.net.ssl.X509TrustManager;
 
 /**
  * Insecure trust manager that does no certificate checking.  Typically
  * used for development and testing use-cases.  Should not be used in
  * production.
  */
-class InsecureTrustManager implements X509TrustManager {
+class InsecureTrustManager extends X509ExtendedTrustManager implements X509TrustManager {
```

Comment:
I'm not sure this was actually necessary but made stepping through things debugger a lot easier. I need to re-visit this before merging.

Reply:
Removed the change.